modules/external-auth-disabling.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-auth-disabling_{context}"]
+= Disabling direct authentication
+
+If necessary, you can disable direct authentication for your cluster and revert back to authenticating with the built-in OpenShift OAuth server.
+
+.Prerequisites
+
+* You have access to the `kubeconfig` file for the cluster.
+
+.Procedure
+
+. Log in to the cluster by using your `kubeconfig` file.
++
+You must use the `kubeconfig` file to configure the direct authentication. If you use a token, the token can become invalid during the process of the procedure.
+
+. Update the authentication configuration to use the built-in OpenShift OAuth server by running the following command:
++
+[source,terminal]
+----
+$ oc patch authentication.config/cluster --type=merge -p='
+spec:
+  type: "" <1>
+  oidcProviders: null <2>
+'
+----
+<1> Sets `type` to `""` to use the built-in OpenShift OAuth server.
```

Comment:
Maybe leave a note that `"IntegratedOAuth"` is equivalent?

Reply:
Added a note!